packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -353,6 +353,53 @@ describe('Right Click Menu integration', () => {
 
       expect(rightClickMenu()).not.toBeNull();
     });
+
+    it('should open and close the context menu using keyboard shortcuts', async () => {
+      const textA = await addText({
+        fontSize: 60,
+        content: '<span style="color: #ff0110">Some Text Element</span>',
+        backgroundColor: { r: 10, g: 0, b: 200 },
+        lineHeight: 1.4,
+        textAlign: 'center',
+        border: {
+          left: 1,
+          right: 1,
+          top: 1,
+          bottom: 1,
+          lockedWidth: true,
+          color: {
+            color: {
+              r: 0,
+              g: 0,
+              b: 0,
+            },
+          },
+        },
+        padding: {
+          vertical: 0,
+          horizontal: 20,
+          locked: true,
+        },
+      });
```

Comment:
I'm sure this is just pulled from the tests around here, but should we try and add the text element through the UI instead just to follow karma best practices?

Reply:
oh yeah good call, can do